### DISCUSSION TITLE
UVTools Exposure Finder strange behaviour at Chitu E10 8K FirmWare 1.0.0 - 1.0.7

### DISCUSSION THREAD

**Discussion by vadimav1** [sn4k3/UVtools#620] [Q&A] [answered]
I have a  Chitu E10 8K FirmWare 1.0.0 - 1.0.7 printer.
I  wish to make 16  tests  which must print simultaneously (together), with exposure from 0.5 to  8 sec.
I used UVTools Exposure Finder.
I multiplyed  standart  UVTools test pattern  16 times and generate a simply short test with growing exposure time from 05 to 8sec. 
I use printer without table, watching process for control at it's screen and measure Z heiht.
But  printer aborted test and  show an error "print error and button 'Shure' ".
Printer stopped to show errors   when i use lift height >=1mm. (mark 2 at Fig. below)
With such bigger value printer worked without showing errors at a touch screen. 
But print process looks like strange. 
Lighting of Every of 16 elements leads to table shift at value of entered layer thickness = 0.1mm  (mark 1), so as a result whole  big layer thickness appeared as 16*layer thickness =1.6mm.  

Question: Is it printer work correcly , or i do not understand some things in this test adjustments? 
Seems that i must type  16 times smaller  value to a layer window,  waiting stepwise-like placement of 16 elements,  but how to be with base layers, they would  become too thin?

![изображение](https://user-images.githubusercontent.com/118778617/206930564-b172adf2-8c07-4da5-bc49-6387ac005648.png)

**Comment by sn4k3** [comment, marked as answer]
Some users is getting problem with some printers, unable to generate multiple exposure tests with new format of CTB Encrypted.
Like Mighty 8K and Sonic Mini 8K are getting same problem that I'm unable to know why without a proper printer to test.

Others:
- https://www.facebook.com/groups/uvtools/posts/817615086014663/
- https://www.facebook.com/groups/uvtools/posts/795122058263966/
- https://www.reddit.com/r/Phrozen/comments/z01qc0/mighty_8k_seems_stuck_at_first_layer_with_uvtools/

**Reply by vadimav1** [reply]
I m  yet even ready now to kick out Chitu motherboard to the trash! This motherboard is drinking a lot of blood for me along 2 months, and I still cannot start to use it! Chitu support appeals that all these things are disabled by firmware, but do not respond on a question "what is a reason of such happening strong restrictioning?"
Can You recommend me  8K 10' mother board which use open codes (G-Code+ZIP) for print-file format? 
May be -  Is it worth to buy some other cheap printer, add precise and rigid Z-positioner and quality paraLED?

**Reply by sn4k3** [reply]
>  Chitu support appeals that all these things are disabled by firmware

Or they have a special flag to turn that ON, I can't debug that because I don't own any CTB printer.
Chitubox pro can slice different parameters per height, one could try it and see if it works and if then get the flag we need.

>  but do not respond on a question 

Do not expect any kind of support about that.

> Can You recommend me 8K 10' mother board which use open codes (G-Code+ZIP) for print-file format?

So far I'm not aware of any, but in the future we can expect the OpenMSLA board.

> Maybe - Is it worth to buy some other cheap printer, add precise and rigid Z-positioner and quality paraLED?

I have an IBEE which is gcode, very rigid machine, but also have it cons. Hardware side is built like a tank. 4K machine. Very good light source. Company support is bad.
In 4k gcode world you also have UniFormation GKone with gcode, also built like a tank and very good machine. (Note: GKtwo is CTB!)

**Reply by vadimav1** [reply]
Thank You for respond. 
As an proposal: we can work together testing different variations of codes  in aim to find out a special flag, which will enable proper working of printer.  As i think  most 8k printers have Chitu E10 motherboard.

**Reply by sn4k3** [reply]
That must be done by user, I have no Chitubox Pro nor printer, so I'm tied.
You can try with UVtools timelapse tool and experiment the two raise methods and test print. If printer raise to top at each **0** mm (every layer) then is a firmware restriction to prevent same layer height's and I'm afraid nothing can be done if so.

**Reply by vadimav1** [reply]
Ok. I understood. Please  tell me more detais about  "UVtools timelapse tool". I can not find out it in menus. Or You mean some another things?

**Reply by sn4k3** [reply]
UVtools -> Tools -> Timelapse.
Use raise each: 0mm

Save and test

**Reply by vadimav1** [reply]
Hello again. It seems,  that Foton Ultra 8K do not have  Chitu E10 8k  motherboard inside. 
The AnyCubic company add to Photon Ultra 8k  10' printer  exposure finder tests and they works.
May be, it's worth to buy AnyCubik 8k and instal it's motherboard instead of chitu?

**Reply by sn4k3** [reply]
All Anycubic use it own board and firmware. The firmware is very bad and not compatible with multi exposure UVtools test nor some other utilities. In general avoid anycubic until they make something usable.

**Reply by vadimav1** [reply]
Are You saying that MB Chitu is less worst thing than MB AnyCubik, although all this MBs- continue to be the wostest in terms of opennes and functionality?   May be uptodate NanoDLP can work with 8k?  What means one can apply with less blood to get G-Code 8K printer?

**Reply by sn4k3** [reply]
Yes, Chitu have the more capable MB right now for 8K printers. Anycubic are bad if you want to use exotic approach or simply the AA. 
If you are regular user that only wants to print out of slicer without any other tune or strategies then it won't matter.
I'm afraid ATM the chitu is the only option for 8K.
You have gcode printers only in 4K ATM.

"Soon" We will have the OpenMSLA board which will be open and give all the features we need. Chitu and others insists in not listen to community and do as they want. The creation of a new fileformat to fit all our needs cost nothing and is easy to do, still they don't care.

Btw have you tried the timelapse tool test?

**Reply by vadimav1** [reply]
I agree. Simply ZIP with enumerated PNGs and + one text file with G-Code (I believe, that OpenMSLA will have the same format. It's very convenient)  is optimal for all, as beginning users, as printing deers, as well as for professors and scientific work.  Probably, Chitu  is trying to stop any user's creative investigation work. 
For example, using simply exposure finder, one can find proper exposure during one 30min printer session. Now, with chitu i must print the same object 30 times separately with a lot of washing, filtering, ...etc.  and it will take about a day itself!

**Reply by sn4k3** [reply]
Yes, they are making SLA very hard which is a non-sense, FDM is miles ahead.
